bin/README.md

```diff
@@ -31,7 +31,7 @@ For a more permanent solution you might have in your `~/.bashrc` or
     NODE_HOME=~/tools/node/current
     ...
 
-    export PATH=~/bin:$SCALR_HOME/bin:$NODE_HOME/bin
+export PATH=~/bin:$SCALR_HOME/bin:$NODE_HOME/bin:$PATH
```

Comment:
The spaces at the beginning of this were meant for making this a markdown code block.